.github/workflows/perf-ci.yml

```diff
@@ -37,6 +37,18 @@ jobs:
       - name: Bench memory usage
         run: cargo bench --bench bench_memory -- --output-format bencher | tee -a output.txt
 
+      - name: Bench network filter matching (flat)
```

Comment:
let's remove this in favor of enabling `flatbuffers-storage` by default (or in the tests above).

Reply:
in the next pr